### DISCUSSION TITLE
How to undo "show less activity like this"?

### DISCUSSION THREAD

**Discussion by J3ekir** [community/community#123181] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I clicked show less activity like this button by mistake on a repo I like. Now it doesn't show the repo's activity on home page. I couldn't find anything about it. The closest thing I found is this answer which is the opposite of what I want.

https://github.com/orgs/community/discussions/30431#discussioncomment-4550989

**Comment by Doruk34** [comment]
You have to click "Customize your pins" when you go down in your profile and pin repos that you want to see.

**Reply by J3ekir** [reply]
I'm not talking about my profile. I'm talking about home page. I don't want to see the repo(s). I want the releases, announcements, etc. in there.
If I star a repo right now, I can see it in there. But I can't see the repo(s) I starred and clicked "show less activity like this" before.

**Comment by J3ekir** [comment, marked as answer]
Seems like it's not possible right now. 

![Screenshot_20240515-003947_Chrome](https://github.com/community/community/assets/53034558/d9daf9ea-7098-4d6b-b47e-bb208c3ccd4e)
